src/app.py

```diff
@@ -9,19 +9,24 @@
 import logging
 
 from controllers.health import health
+from exception_handler import exceptions, exception_handlers
+
+
 
 logging.basicConfig(level=logging.DEBUG)
 PORT = 5001
 HOST = "0.0.0.0"
 
 # For more configuration https://swagger.io/docs/open-source-tools/swagger-ui/usage/configuration/
 options = {
-    "swagger_ui": False,
+    "swagger_ui": True,
     "swagger_ui_config": {
         "displayOperationId": False
     }
 }
 
+exception_codes = [401,403,405,500]
```

Comment:
make this a dictionary, and also add the name to it i.e. 401: Unauthorised, makes for better reading.